de.metas.business/src/main/java-legacy/org/compiere/model/MInOut.java

```diff
@@ -21,12 +21,12 @@
 import java.sql.ResultSet;
```

Comment:
i would avoid formating/cleaning up code for huge legacy classes like this one because
* makes the review much more harder
* makes integration of other PRs much more harder (think that i have a PR which changes a line in this class....)

Reply:
* I modified yesterday a setting from eclipse and that caused all that. (in the idea of improving code a bit, at least visual)
* I'll take out that setting and see what can I do to with this